src/Panel.tsx

```diff
@@ -33,23 +33,37 @@ function liveRegionChangesReducer(state: State, action: Action) {
       };
     case "clear":
       return INITIAL_STATE;
+    // @ts-expect-error
+    case "unknown_type":
+      return state;
```

Comment:
interesting. in which cases would the `"unknown_type"` be passed in. would this be passed in deliberately or is catching some React/TS magic?

Reply:
It would never be passed in in this case being that you're manually dispatching only known actions and not exposing any type of api, but this just helps you make sure your cases match being that we're not using `ACTION_TYPES.CLEAR` type of things (which this could be changed).

Reply:
that makes sense. yeah, constants are being used in other places in the apps. felt a little overkill in this single component reducer. but yeah, that could definitely change as complexity increases.